README.md

```diff
@@ -12,15 +12,15 @@ To install it, please run:
 python3 -m pip install web.py==0.62
 ```
 
-If you are still using Python 2.7, then please use web.py version 0.51
-which is intended to be our last release that supports Python 2. 
+If you are using Python 2.7, then please use web.py version 0.51
```

Comment:
```suggestion
If you are still using Python 2.7, then please use web.py version 0.51
```
The use of `still` was quite deliberate and is even more appropriate today then when it was first written.